clrfeatures.cmake

```diff
@@ -3,23 +3,13 @@ if(CLR_CMAKE_TARGET_TIZEN_LINUX)
 endif()
 
 if(NOT DEFINED FEATURE_EVENT_TRACE)
-  if (WIN32)
```

Comment:
I think that this is a fair change to make even on OSX where we're just going to use EventPipe.  It is also reasonable to look at merging these two together since we're basically going to say that we have a default pipe which is EventPipe on all platforms.  I would recommend filing an issue for this and coming back to it, since we should aim to get this merged soon and it's not critical.